intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -496,6 +496,71 @@ \subsection{Getting module details}
 If you're not sure what all of this means: don't worry, you don't have to know;
 just load the module and try to use the software.
 
+\section{Getting system information about the cluster}
+
+\subsection{Checking the general status of the cluster}
+
+To check the general system state, check \url{https://www.vscentrum.be/en/user-portal/system-status}.
+This has information about scheduled downtime, status of the system, \ldots
+
+To check how much jobs are running in what queues, you can use the \lstinline|qstat -q|
+command:
+
+\begin{prompt}
+%\shellcmd{qstat -q}%
+Queue            Memory CPU Time Walltime Node  Run Que Lm  State
+---------------- ------ -------- -------- ----  --- --- --  -----
+default            --      --       --      --    0   0 --   E R
+q72h               --      --    72:00:00   --    0   0 --   E R
+long               --      --    72:00:00   --  316  77 --   E R
+short              --      --    11:59:59   --   21   4 --   E R
+q1h                --      --    01:00:00   --    0   1 --   E R
+q24h               --      --    24:00:00   --    0   0 --   E R
+                                               ----- -----
+                                                337  82
+\end{prompt}
+
+Here, there are 361 jobs running on the ``long'' queue, and 77 jobs queued. We
+can also see that the ``long'' queue allows a maximum wall time of 72 hours.
+
+\ifgent
+\subsection{Getting cluster state}
+
+You can check \url{http://hpc.ugent.be/clusterstate} to see information about
+the clusters: you can see the nodes that are down, free, partially filled with jobs,
+completely filled with jobs, \ldots.
+
+You can also get this information with a command: \lstinline|pbsmon|:
+
+\begin{prompt}
+%\shellcmd{module swap cluster/phanpy}%
+%\shellcmd{pbsmon}%
+2301 2302 2303 2304 2305 2306 2307
+    J    x    x    J    J    x    J
+
+ 2308 2309 2310 2311 2312 2313 2314
+    J    J    J    J    J    x    J
+
+ 2315 2316
+    J    J
+
+   J full                 : 12  |   X down                 : 0   |
+   j partial              : 0   |   x down_on_error        : 4   |
+   _ free                 : 0   |   . offline              : 0   |
+                                |   _ offline_idle         : 0   |
+                                |   o other                : 0   |
```

Comment:
Please edit this to have a single node offline, a single node down, and a couple of nodes free & partially free.